mobx_codegen/test/data/valid_annotated_store_referencing_store_input.dart

```diff
@@ -1,17 +1,17 @@
 library generator_sample;
 
-import 'dart:ui' as ui;
-
 import 'package:mobx/mobx.dart';
 
+import 'dart:ui';
```

Comment:
I think we also need  a test which includes the prefix and ensure that works too. Not sure if we can have a hybrid approach?

Reply:
Right, so it won't work anymore. In fact, prefixes only ever worked when the types were detected as dynamic, like with dart:ui types, but not in other situations.

You can see for yourself. If you open up `valid_input.dart` in the `mobx_codegen` tests, and import dart:core like this:

```dart
import 'dart:core' as c;

abstract class UserBase with Store {
  UserBase(this.id);

  final c.int id;

  @observable
  c.String firstName = 'Jane';

  @observable
  c.String lastName = 'Doe';

  @computed
  c.String get fullName => '$firstName $lastName';

...
```
You'll see that `valid_output.dart` doesn't require changing to get the test to pass, yet it contains unprefixed core types. In a real situation it wouldn't compile properly alongside the input.

I propose we fix prefixes across the board.

Reply:
When you say fix across the board, are you referring to removing the prefix everywhere?

Reply:
No, adding support for them.

I've pushed a new version that adds support, as well as a test.